### PR TITLE
INTMDB-771: [CFN-PI] Add RELEASE.md to the CDK repository

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ INTMDB-771

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] I have tested the CDK constructor in a CFN stack. See [TESTING.md](../TESTING.md)

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Releasing
+
+
+## Stable release
+
+Stable releases are managed by the [Release GitHub Action](https://github.com/mongodb/awscdk-resources-mongodbatlas/actions/workflows/release.yml). You can release a new CDK version by manually trigger the action.
```
[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply RELEASING.md:6]: Based on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).

 See [releases.html#breaking-changes](https://projen.io/releases.html#breaking-changes)

[the remark this answers is not in the document]
Comment (maastha) [reply RELEASING.md:6]: Should we add a link to this to RELEASE.md in that case?

[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply RELEASING.md:6]: Updated

**Review by maastha — APPROVED** [review]
LGTM

**Pull request merged**